app/Http/Controllers/AppController.php

```diff
@@ -315,16 +323,16 @@ public function app_post(Request $request){
             return response($error_response);
         }
 
-        if(!$request->has('content')){
+        if(!$request->has('post_content')){
```

Comment:
This requires change in app as app consumes it

Reply:
ok. Reverted it back